pkg/crd/kubermatic/v1/settings.go

```diff
@@ -73,6 +75,19 @@ type CleanupOptions struct {
 	Enforced bool
 }
 
+type MachineDeploymentVMResourceQuota struct {
```

Comment:
Shouldn't this follow the same resources handling and types as k8s?

Reply:
no, we are going to use common parameters for all providers such as a number of vCPU and Memory. This is what we have from the provider clients and will be easy to use to filter out VMs

Reply:
Alright, I mean as long as the user can actually override those then I think we are on the good side here.